Description: fix include paths
 When using the pkg-config file of libbibutils, the includes are to be searched
 for in bibutils/
Author: Joachim Breitner <nomeata@debian.org>

--- a/src/Text/Bibutils.hsc
+++ b/src/Text/Bibutils.hsc
@@ -402,7 +402,7 @@
 
 newtype FormatOpt = FormatOpt { unFormatOpt :: CInt }
 
-#include "bibutils.h"
+#include "bibutils/bibutils.h"
 
 #{enum FormatOpt, FormatOpt
  , bibout_finalcomma = BIBL_FORMAT_BIBOUT_FINALCOMMA
@@ -437,8 +437,8 @@
 combineFormatOpts :: [FormatOpt] -> FormatOpt
 combineFormatOpts = FormatOpt . foldr ((.|.) . unFormatOpt) 0
 
-#include "bibutils.h"
-#include "bibl.h"
+#include "bibutils/bibutils.h"
+#include "bibutils/bibl.h"
 #let alignment t = "%lu", (unsigned long)offsetof(struct {char x__; t (y__); }, y__)
 
 foreign import ccall unsafe "bibl_init"
